contrib/container_images/build_and_publish_rust_image.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+PODMAN=${PODMAN:-podman}
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+DOCKERFILE="$(dirname "${BASH_SOURCE[0]}")/Dockerfile.Rust"
+
+# get the tag from the Dockerfile so we do not duplicate it
+TAG=$(awk -F':' '/FROM/{print $NF}' $DOCKERFILE)
```

Comment:
What would you think of `if [[ -z "$TAG" ]]; then echo "FOO! empty tag"; exit 1; fi` or equivalent?

Reply:
yes sure